handlers/delete_lease_test.go

```diff
@@ -14,28 +15,36 @@ import (
 )
 
 func TestDeleteLeaseNoToken(t *testing.T) {
+	os.Setenv("AUTH_TOKEN", "some awesome token")
+	defer os.Clearenv()
 	getterUpdater := newFakeServiceGetterUpdater(nil, nil, nil, nil)
 	nsListerDeleter := newFakeNamespaceListerDeleter(nil, nil, nil)
 	hdl := DeleteLease(getterUpdater, "claimer", nsListerDeleter)
 	req, err := http.NewRequest("DELETE", "/lease", nil)
+	req.Header.Set("Authorization", "some awesome token")
 	assert.NoErr(t, err)
 	res := httptest.NewRecorder()
 	hdl.ServeHTTP(res, req)
 	assert.Equal(t, res.Code, http.StatusBadRequest, "response code")
 }
 
 func TestDeleteLeaseInvalidLeaseToken(t *testing.T) {
+	os.Setenv("AUTH_TOKEN", "some awesome token")
+	defer os.Clearenv()
 	getterUpdater := newFakeServiceGetterUpdater(nil, nil, nil, nil)
 	nsListerDeleter := newFakeNamespaceListerDeleter(nil, nil, nil)
 	hdl := DeleteLease(getterUpdater, "claimer", nsListerDeleter)
 	req, err := http.NewRequest("DELETE", "/lease/abcd", nil)
+	req.Header.Set("Authorization", "some awesome token")
 	assert.NoErr(t, err)
 	res := httptest.NewRecorder()
 	hdl.ServeHTTP(res, req)
 	assert.Equal(t, res.Code, http.StatusBadRequest, "response code")
 }
 
 func TestDeleteLeaseInvalidAnnotations(t *testing.T) {
+	os.Setenv("AUTH_TOKEN", "some awesome token")
+	defer os.Clearenv()
```

Comment:
I don't think we need this env var set anymore. same with the other similar env vars set in other tests